README.md

```diff
@@ -87,15 +87,15 @@ For doing so, `rmw_fastrtps` locates profiles in the XML based on topic names ab
 
 ##### Creating publishers/subscriptions with different profiles
 
-To configure a publisher/subscription, define a `<publisher>`/`<subscriber>` profile with attribute `profile_name=topic_name`.
+To configure a publisher/subscription, define a `<publisher>`/`<subscriber>` profile with attribute `profile_name=topic_name`, where topic name is the name of the topic before mangling, i.e. the topic name used to create the publisher/subscription on `rclcpp`/`rclpy`.
```

Comment:
@EduPonz consider not mentioning client libraries directly to stay general (as examples at most)

Reply:
Done in 4909e2b